recipes/pinocchio/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake .. \
+      -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH=$PREFIX -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=lib
+
+make
```

Comment:
Can multiple cores be used?
```suggestion
make -j${CPU_COUNT}
```

Reply:
Out of curiosity how many cores are available on Azure Pipelines?

Reply:
I can't remember exactly but it's either 4 or 8 and I think macOS has more than Linux.